### PR TITLE
Fixing scylla manager master

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fgelcer — APPROVED** [review]
LGTM

File: sdcm/utils/prepare_region.py [hunk 1]
```diff
@@ -309,14 +309,50 @@ def create_security_group(self):
...
+                                      'Description': 'Allow Scylla Manager Agent Prometheus API for ALL'}],
+                        "Ipv6Ranges": [{'CidrIpv6': '::/0',
+                                        'Description': 'Allow Scylla Manager Agent Prometheus API for ALL'}]
+                    },
+                    {
+                        "FromPort": 5112,
+                        "ToPort": 5112,
+                        "IpProtocol": "tcp",
+                        "IpRanges": [{'CidrIp': '0.0.0.0/0',
+                                      'Description': 'Allow Scylla Manager pprof Debug For ALL'}],
```
Comment (fruch) [inline sdcm/utils/prepare_region.py:353]: we ever used this port in SCT ?

**Comment by fruch** [comment]
@ShlomiBalalis backport labals are missing

**Review by fruch — APPROVED** [review]
LGTM,
but @bentsi pay attention to the new port being opened, I don't know how those are getting updated

**Comment by fruch** [comment]
@ShlomiBalalis 
lint of the test cases failed:
https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-2549/1/execution/node/41/log/

I've rerun it.

But I would recommend removing/replacing `https://repositories.scylladb.com/scylla/repo/qa-test/centos/scylladb-manager-2.0.repo` or any other urls that go via the lambda function, it not very stable yet.

File: sdcm/utils/prepare_region.py (hunk 1 above)
Comment (ShlomiBalalis) [reply sdcm/utils/prepare_region.py:353]: SCT doesn't, but the manager does

**Comment by ShlomiBalalis** [comment]
> LGTM,
> but @bentsi pay attention to the new port being opened, I don't know how those are getting updated

@fruch I updated the existing security groups manually

**Comment by bentsi** [comment]
@ShlomiBalalis what is the status of this? Why is it draft?

**Comment by ShlomiBalalis** [comment]
> @ShlomiBalalis what is the status of this? Why is it draft?

I had some issues with the upgrade test, I solved them.

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
@ShlomiBalalis once it's passing on master, we need to create branch manager-2.2

**Pull request merged**